pypeit/par/pypeitpar.py

```diff
@@ -2698,8 +2698,8 @@ def __init__(self, reference=None, method=None, echelle=None, ech_nspec_coeff=No
         defaults['rms_threshold'] = 0.15
         dtypes['rms_threshold'] = [float, list, np.ndarray]
         descr['rms_threshold'] = 'Minimum RMS for keeping a slit/order solution. This can be a ' \
-                                 'single number or a list/array providing the value for each slit. ' \
-                                 'Only used if ``method`` is either \'holy-grail\' or \'reidentify\''
+                                 'single number or a list/array providing the value for each order. ' \
+                                 'Only used if ``method`` for \'reidentify\' and echelle spectrographs'
```

Comment:
Please elaborate the docs here and explain that t is now in units of FWHM (is it?). Can you also indicate the units in the docs either way (pixels?).